plugins/zippyshare.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# https://github.com/Sorrow446/ZS-DL
+# plugin by @aryanvikash
+
+import os
+import re
+import sys
+import json
+import time
+import requests
+try:
+    from urllib.parse import unquote
+except ImportError:
+    from urllib import unquote
+import asyncio
+from userge import userge, Message
+
+
+@userge.on_cmd("zippy", about={
+    'description': "generate Direct link of zippyshare url",
+    'usage': "{tr}zippy : [Zippyshare Link ]",
+    'examples': "{tr}zippy https://www10.zippyshare.com/v/dyh988sh/file.html"}, del_pre=True)
+async def zippyshare(m: Message):
+    url = m.filtered_input_str
+    loop = asyncio.get_event_loop()
+    await m.edit("Generating url ....")
+    try:
+        direct_url, fname = await loop.run_in_executor(None, generate_zippylink, url)
```

Comment:
great, can you use pool instead of this

Reply:
Don't know about pool .. will you please Do it :)

Reply:
ok, thanks for this plugin

Reply:
Don't know about pool . will you please do it :)